### PR TITLE
PFS and integrity-only ciphersuites

### LINKED ISSUE #43: RFC 9150 - Integrity Only Ciphersuites for TLS
Not all suites offer pfs.  See RFC 9150.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #43

### CODE REVIEW

File: draft-ietf-uta-tls13-iot-profile.md [hunk 1]
```diff
@@ -224,10 +224,16 @@ protocol. Additionally, the work on Compact TLS (cTLS) {{?I-D.ietf-tls-ctls}} ha
 a step further by utilizing out-of-band knowledge between the communication parties to reduce
 the amount of data to be transmitted at each individual handshake, among applying other techniques.
 
-# Perfect Forward Secrecy
+# Forward Secrecy
 
-TLS 1.3 allows the use of PFS with all ciphersuites since the support for it is
-negotiated independently.
+RFC8446 has removed Static RSA and Diffie-Hellman cipher suites, therefore all public-key-based key exchange mechanisms available in TLS 1.3 provide forward secrecy.
```
Comment (hannestschofenig) [inline draft-ietf-uta-tls13-iot-profile.md:229]:
```suggestion
RFC 8446 removed cipher suites that did not provide forward secrecy.
```

File: draft-ietf-uta-tls13-iot-profile.md [hunk 2]
```diff
@@ -224,10 +224,16 @@ protocol. Additionally, the work on Compact TLS (cTLS) {{?I-D.ietf-tls-ctls}} ha
...
+
+Pre-shared keys (PSKs) can be used with (EC)DHE key exchange to provide forward secrecy or can be used alone, at the cost of losing forward secrecy for the application data.
+
+# Authentication and Integrity-only Cipher Suites
+
+For a few, very specific Industrial IoT use cases {{?RFC9150}} defines two cipher suites that provide data authenticity, but not data confidentiality.
```
Comment (hannestschofenig) [inline draft-ietf-uta-tls13-iot-profile.md:235]:
```suggestion
For a few Industrial IoT use cases, such as railway scenarios and power system automation, {{?RFC9150}} defines two cipher suites that provide data authenticity, but not data confidentiality.
```

File: draft-ietf-uta-tls13-iot-profile.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-ietf-uta-tls13-iot-profile.md:229]: I framed the section as:

1. Public-key-based ciphersuite, vs
2. PSK-based ciphersuite

and their FS properties.

I believe the suggestion drops some of the semantics.

Comment (hannestschofenig) [inline draft-ietf-uta-tls13-iot-profile.md:229]:
```suggestion
RFC 8446 has removed Static RSA and Static Diffie-Hellman cipher suites, therefore all public-key-based key exchange mechanisms available in TLS 1.3 provide forward secrecy.
```

**Pull request merged**